src/PoolManager.sol

```diff
@@ -269,19 +270,19 @@ contract PoolManager is IPoolManager, Fees, NoDelegateCall, ERC6909Claims {
         uint256 reservesBefore = reservesOf[currency];
         reservesOf[currency] = currency.balanceOfSelf();
         paid = reservesOf[currency] - reservesBefore;
-        // subtraction must be safe
-        _accountDelta(currency, -(paid.toInt128()));
+        _accountDelta(currency, paid.toInt128());
     }
 
     /// @inheritdoc IPoolManager
     function mint(address to, uint256 id, uint256 amount) external override noDelegateCall isLocked {
-        _accountDelta(CurrencyLibrary.fromId(id), amount.toInt128());
+        // flipping a postive number in an int is always safe
```

Comment:
same here. "Must be safe" (the old comment)is less confusing if we do want to keep a comment.